visyn_core/security/store/alb_security_store.py

```diff
@@ -9,25 +14,44 @@
 _log = logging.getLogger(__name__)
 
 
+@lru_cache
+def _get_public_key(region: str, kid: str) -> str:
+    return httpx.get(f"https://public-keys.auth.elb.{region}.amazonaws.com/{kid}").text
```

Comment:
Initially, it was unclear to me what `kid` is. I would expand it:

```suggestion
def _get_public_key(region: str, key_id: str) -> str:
    return httpx.get(f"https://public-keys.auth.elb.{region}.amazonaws.com/{key_id}").text
```

Reply:
Done, renamed it and added it to the comment.